widget/list.go

```diff
@@ -86,6 +89,23 @@ func (l *List) CreateRenderer() fyne.WidgetRenderer {
 	return newListRenderer(objects, l, l.scroller, layout)
 }
 
+// FocusGained is called after this List has gained focus.
+//
+// Implements: fyne.Focusable
+func (l *List) FocusGained() {
```

Comment:
More of a general comment than specific to this PR, but is it possible with the focus system for a user to override a Focusable widget and effectively make it non-focusable by calling `FocusNext()` in the `FocusGained` implementation? Say if there were a use case where it really was easier to handle focus in the row widgets rather than the list itself

Reply:
If you know what "next" is you can ask the canvas to focus it in your "FocusGained", yes.

Reply:
the focus API also has `.FocusNext()` so should be good in any case